Source/Events/Store/Converters/EventContentSerializer.cs

```diff
@@ -12,14 +12,17 @@ namespace Dolittle.SDK.Events.Store.Converters
     public class EventContentSerializer : ISerializeEventContent
     {
         readonly IEventTypes _eventTypes;
+        readonly Func<JsonSerializerSettings> _jsonSerializerSettingsProvider;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="EventContentSerializer"/> class.
         /// </summary>
         /// <param name="eventTypes"><see cref="IEventTypes"/> for mapping types and artifacts.</param>
-        public EventContentSerializer(IEventTypes eventTypes)
+        /// <param name="jsonSerializerSettingsProvider"><see cref="Func{T}"/> that provides <see cref="JsonSerializerSettings"/>.</param>
+        public EventContentSerializer(IEventTypes eventTypes, Func<JsonSerializerSettings> jsonSerializerSettingsProvider = null)
```

Comment:
This cannot have a default value now :)
```suggestion
        public EventContentSerializer(IEventTypes eventTypes, Func<JsonSerializerSettings> jsonSerializerSettingsProvider)
```